tests/csm/rest/test_capacity_usage.py

```diff
@@ -126,6 +135,47 @@ def teardown_method(self):
             assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} way")
             self.log.info("Successfully restored pod by %s way", self.restore_method)
 
+    def s3_ios(self,
```

Comment:
There are multiple problems with this func
1. Duplicate func
2. It doesnt uses the access key and secret key defined in setup class

best would be to directly call
s3bench.s3bench( self.akey,self.skey....)

You can access num_clients, num_sample, obj_name_pref from test_cfg

Reply:
done